contracts/Agreement.sol

```diff
@@ -1,104 +1,156 @@
- *
- */
+* 
+* ██████████████████████▀████████████████████
+* █▄─▄▄─█▄─▄─▀█─▄▄─█─▄▄▄▄███▄─▄▄▀██▀▄─██─▄▄─█
+* ██─▄█▀██─▄─▀█─██─█─██▄─████─██─██─▀─██─██─█
+* ▀▄▄▄▄▄▀▄▄▄▄▀▀▄▄▄▄▀▄▄▄▄▄▀▀▀▄▄▄▄▀▀▄▄▀▄▄▀▄▄▄▄▀
+*
+* Please read this participation ("agreement") 
+* carefully before confirming your intent to 
+* be bound by it and participating in the
+* OG DAO. This agreement includes the terms
+* of participation in the OG DAO. You 
+* understand, agree and confirm that:
+*
+* 1.) The OG DAO is an experiment in the 
+*     field of decentralized governance
+*     structures, in which participation 
+*     is entirely at your own risk.
+* 2.) This agreement has legal consequences 
+*     and by entering into this agreement 
+*     you release all rights, claims, or 
+*     other causes of action whether in 
+*     equity or law you may have against 
+*     OG DAO service providers or other
+*     OG DAO participants. You also agree to
+*     waive and limit any potential liability
+*     of OG DAO service providers or other 
+*     OG DAO participants. 
+* 3.) You are sophisticated and have 
+*     sufficient technical understanding of
+*     the functionality, usage, storage, 
+*     transmission mechanisms, and intricacies
+*     associated with cryptographic tokens, 
+*     token storage facilities (including 
+*     wallets), blockchain technology, and 
+*     blockchain-based software systems.
+* 
+* 
+* There are 2 options:
+* 1.) I, the owner of this address, 
+*     Opt-out completely from the OG DAO.
+* 
+* 2.) I, the owner of this address, Opt-in
+*     to the OG DAO. I have read and agreed 
+*     to the participation agreement below.
+* 
+* 
+* If you do not opt-in or out, you are still a 
+* member of the OG DAO. You still have the same 
+* voting power, however you will not receive 
+* airdrops, payments, or other benefits. This is
+* to protect the OG DAO, it’s members, and assets.
+* 
+*/
+
 
 import "@openzeppelin/contracts/access/AccessControl.sol";
 import "@openzeppelin/contracts/access/Ownable.sol";
-import "hardhat/console.sol";
 
-contract Agreement is AccessControl, Ownable {
+contract OGAgreement is AccessControl, Ownable {
     uint256   public totalAccounts = 0;
-    address[] public optedInAccounts;
-    address[] public optedOutAccounts;
+    address[] public optInAccounts;
+    address[] public optOutAccounts;
+    bool      public contractActive;
 
     mapping (address => bool) public optIn;
     mapping (address => bool) public optOut;
 
+    // @dev Create the community role with the contract owner as a member.
     constructor() {
-      _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
-      _setupRole("OG", msg.sender);
-    }
-
-    // @dev Restricted to members of the community.
-    modifier onlyMember() {
```

Comment:
I don't remember deleting this...I don't think I have the latest version. It still says EBOG in the role